### PR TITLE
Improve the messaging for MP schema checking during non-release PRs

### PR DESCRIPTION
**Pull request by orta**
+ and add danger messaging on release PRs

Screen Shot 2019-03-26 at 10 24 04 AM

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -273,6 +273,7 @@
   },
   "devDependencies": {
     "@babel/plugin-transform-modules-commonjs": "^7.2.0",
+    "@graphql-inspector/core": "^1.14.0",
```
Comment (orta) [inline package.json]: I added this to Peril a few weeks back, it provides a way for us to take the generic changes between two GraphQL schemas and makes human readable messages from it.

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>The staging version of Metaphysics has breaking changes, this might not affect your build, but could mean your queries don't work the way you expect. See below for breaking changes:</td>
    </tr>
  </tbody>
</table>


- :warning: Type 'AnalyticsPricingContextDimensionsEnum' was removed
- :warning: Field 'analyticsPricingContext' was removed from object type 'Query'
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against c1aefbe012ae1ac37a55293a06522d8195007b21
</p>

**Comment by zephraph** [comment]
So good <3

**Comment by mzikherman** [comment]
Nice.

**Comment by joeyAghion** [comment]
Can you comment about what effective change this makes? I think the new check only runs on PRs, but it's hard to parse the YAML.

Will breaking changes be allowed to deploy to staging? Will production's check remain? (Is it weird to have the PR and release checks driven from different places?)
